backend/routes.py

```
@@ -1,15 +1,95 @@
 import json
 from uuid import uuid4
 from fastapi import BackgroundTasks
 import asyncio
-from datetime import datetime
+from datetime import datetime, timedelta
 from config import settings
 from utils.file_validator import FileValidator, FileValidationError
 from utils.error_messages import ErrorMessages
+from utils.content_cache import get_content_cache
+from typing import Dict, Any, Optional
+from utils.secure_temp_files import SecureTempFile, get_memory_storage, secure_temp_file
```

Ruff (0.12.2)

`utils.secure_temp_files.secure_temp_file` imported but unused

Remove unused import: `utils.secure_temp_files.secure_temp_file`

(F401)